neo/Helper.cs

```diff
@@ -320,5 +320,18 @@ public static IConfigurationRoot LoadConfig(string config)
                 .AddJsonFile(configFile, true)
                 .Build();
         }
+
+        public static T StrictEnum<T>(byte value) where T : Enum
+        {
+            T state = (T)Enum.ToObject(typeof(T), value);
+            if (!EnumValueCache<T>.definedValues.Contains(state))
+                throw new FormatException();
```

Comment:
I think we should throw `ArgumentOutOfRangeException` here. But the caller may want to throw `FormatException`. So the best idea is to throw exception in the caller.